### PR TITLE
Add Patient_tribal_affiliation and Patient_preferred_language to TestEventExport

### LINKED ISSUE #1799: Send Patient Preferred Language and Tribal Affiliation to ReportStream
We capture patient preferred language right now, but we don't share it with public health departments. We should send it along with test results when we have it.

@jimduff-usds FYI, we will need guidance on where to put patient preferred language in the data we send to you

### Tasks

- [ ] Figure out how to put this in the schema for reportstream

- [ ] Add it to the schema

### PR DESCRIPTION
**Pull request by nullflux**
## Related Issue or Background Info

- Resolves #1799
- Note that this relies on #2397, #2398, #2399 

## Changes Proposed

- Add `Patient_tribal_affiliation` to TestEventExport

### UI
- [x] Any changes to the UI/UX are approved by design 
- [x] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

File: backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java [hunk 1]
```diff
@@ -35,8 +35,8 @@ void serialization() {
     TestResult r = TestResult.NEGATIVE;
     Date d = new Date();
 
-    Person p1 = _dataFactory.createMinimalPerson(o);
-    Person p2 = _dataFactory.createMinimalPerson(o);
+    Person p1 = _dataFactory.createFullPersonWithTelephone(o, "5168675309");
```
Comment (nullflux) [inline backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java:38]: Note that this change significantly increases the diff to this serialization test, but is necessary to get adequate coverage on the new fields. I've manually verified that these changes are as-expected

File: backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java [hunk 2]
```diff
@@ -83,23 +102,25 @@ void serialization() {
...
             + date
-            + "\",\"54321-BOOM\",\"Washington\",\"Washington\",\"facility@test.com\",\"Imaginary Site\",\"555-867-5309\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"DOOOOOOM\",\"Washington\",\"Washington\",\"Doctor\",\"Doom\",\"800-555-1212\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"The Mall\",\"\",\""
+            + "\",\"54321-BOOM\",\"Washington\",\"Washington\",\"facility@test.com\",\"Imaginary Site\",\"555-867-5309\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"DOOOOOOM\",\"Washington\",\"Washington\",\"Doctor\",\"Doom\",\"800-555-1212\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"The Mall\",\"18990510\",\""
             + patientId2
-            + "\",\"\",\"\",\"\",\"U\",\"John\",\"U\",\"Boddie\",\"Brown\",\"503-867-5309\",\"UNK\",\"STAFF\",\"\",\"\",\"\",\"Jr.\",\"\",\"P\",\"UNK\",\""
+            + "\",\"Washington\",\"Washington\",\"\",\"N\",\"Fred\",\"M\",\"Astaire\",\"\",\"5168675309\",\"Spanish\",\"2106-3\",\"RESIDENT\",\"DC\",\"736 Jackson PI NW\",\"\",\"\",\"123\",\"20503\",\"P\",\"N\",\"" // "\",\"\",\"\",\"\",\"U\",\"John\",\"U\",\"Boddie\",\"Brown\",\"503-867-5309\",\"UNK\",\"STAFF\",\"\",\"\",\"\",\"Jr.\",\"\",\"P\",\"UNK\",\""
```
Comment (nathancrtr) [inline backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java:115]: Stray comment?

**Review by nathancrtr — APPROVED** [review]
Ship it! :shipit:

File: backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java (hunk 2 above)
Comment (nullflux) [reply backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java:115]: Ah yeah, gross - this is from when I was adjusting it

Comment (nullflux) [reply backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java]: Fixed in 7aa82a3

**Pull request merged**